include/seahorn/Expr/ExprOpGate.hh

```diff
@@ -20,9 +20,9 @@ struct GateOp : public expr::Operator {
 
 /// an output gate
 NOP(OUT_G, "OUT_G", PREFIX, GateOp)
```

Comment:
Gates are just mutable boolean operators. Their inputs are boolean and their outputs are boolean.

an output gate `OUT_G` marks an output of a function. Its input is a Boolean and output a Boolean